.github/workflows/release_prepare.yml

```diff
@@ -9,7 +9,7 @@ env:
   GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 jobs:
   fetch-versions:
-    if: github.event.created && github.repository == 'zenml-io/zenml'
+    if: github.repository == 'zenml-io/zenml'
```

Comment:
Isn't this whole workflow problematic if it runs multiple times? Appending to the release notes etc?

Reply:
I disabled the prepare-changes step for subsequent code change pushes. This should take care of any problems. Better than to have to delete/recreate the branch.

Reply:
I actually had to iterate through the solution a bit to get it to work, but I think it's good now.